service/league_service/league_rater.py

```diff
@@ -18,4 +18,68 @@ def rate(
         outcome: GameOutcome,
         player_rating: Rating,
     ):
+        boost = 0
+        reduction = 0
+        rating = player_rating[0] - 3 * player_rating[1]
+        player_div = cls._get_player_division(current_score.division_id, league)
+        # Highest divison has lowest id
+        lowest_id, highest_id = cls._get_division_id_range()
+
+        if current_score.game_count >= 10:
+            if current_score.division_id is None:
+                return cls._do_placement(league, current_score, rating)
+
+            if rating > player_div.max_rating:
+                boost = 1
```

Comment:
This `1` and the reduction `1` could also be made a config parameter.

Reply:
Done